src/sentry/static/sentry/app/components/organizations/homeSidebar.jsx

```diff
@@ -34,17 +36,18 @@ const OrgSettingsMenu = ({access, org, features}) => {
     <div>
       <h6 className="nav-header with-divider">{t('Manage')}</h6>
       <ul className="nav nav-stacked">
-        {access.has('org:read') && (
-          <ListLink to={`${pathPrefix}/members/`}>
-            {t('Members')}&nbsp;
-            {access.has('org:write') &&
-              org.pendingAccessRequests > 0 && (
-                <span className="badge" style={{marginLeft: 5}}>
-                  {org.pendingAccessRequests}
-                </span>
-              )}
-          </ListLink>
-        )}
+        {access.has('org:read') &&
+          access.has('member:read') && (
+            <ListLink to={`${pathPrefix}/members/`}>
+              {t('Members')}&nbsp;
+              {access.has('org:write') &&
```

Comment:
Not really related to this PR but might read nicer to extract the many access / features checks from this template